### ISSUE TITLE
如何屏蔽训练过程中间信息的输出

### ISSUE THREAD

**Issue by shingstone-lu** [PaddlePaddle/Paddle#32956]
win10操作系统
python3.7.0-amd64
cuda_10.2.89_441.22_win10
cudnn-10.2-windows10-x64-v8.0.2.39
paddlepaddle_gpu-2.0.2-cp37-cp37m-win_amd64
以上是软件配置。程序代码如下：
import paddle
import numpy as np
data = np.random.uniform(-1, 1, [2, 2]).astype('float32')
print('data:')
print(data)
data = paddle.to_tensor(data)
# create Sequential with iterable Layers
model1 = paddle.nn.Sequential( paddle.nn.Linear(2, 1), paddle.nn.Linear(1, 2))
print('parameter:')
print(model1[0].parameters())
# access the first layer
print(model1[1].parameters())
# access the first layer
res1 = model1(data)
# sequential execution
print('result:')
print(res1)
程序在运行时会输出很多中间信息，像下图中以“I0518”开头的行，如何屏蔽这些信息？
这些信息已经将程序主动输出的训练进度完全淹没了。
不知道怎么添加图像，拷贝了2段输出：

第1段：
WARNING: Logging before InitGoogleLogging() is written to STDERR
I0518 15:32:32.069051  2980 pybind.cc:346] py_func has no kernels, skip
I0518 15:32:32.070046  2980 pybind.cc:346] recurrent has no kernels, skip
I0518 15:32:32.070046  2980 pybind.cc:346] while has no kernels, skip
I0518 15:32:32.070046  2980 pybind.cc:346] conditional_block has no kernels, skip
I0518 15:32:32.076022  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_free_idle_chunk"
I0518 15:32:32.076022  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_free_when_no_cache_hit"
I0518 15:32:32.076022  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_eager_delete_tensor_gb"
I0518 15:32:32.076022  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_enable_parallel_graph"
I0518 15:32:32.077030  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_allocator_strategy"
I0518 15:32:32.077030  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_use_system_allocator"
I0518 15:32:32.077030  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_check_nan_inf"
I0518 15:32:32.077030  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_call_stack_level"
I0518 15:32:32.077030  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_sort_sum_gradient"
I0518 15:32:32.077030  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_cpu_deterministic"
I0518 15:32:32.077030  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_enable_rpc_profiler"
I0518 15:32:32.077030  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_multiple_of_cupti_buffer_size"
I0518 15:32:32.078004  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_reader_queue_speed_test_mode"
I0518 15:32:32.078004  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_pe_profile_fname"
I0518 15:32:32.078004  2980 global_value_getter_setter.cc:258] Get substring: "FLAGS_print_sub_graph_dir"

第2段：
I0518 15:32:40.321995  2980 tracer.cc:189] No Grad to track for Op: gaussian_random
I0518 15:32:40.321995  2980 imperative.cc:543] Init VarBase
I0518 15:32:40.321995  2980 variable_wrapper.h:159] The tensor of variable conv2d_1.b_0 is not initialized
I0518 15:32:40.322993  2980 variable_wrapper.h:159] The tensor of variable conv2d_1.b_0 is not initialized
I0518 15:32:40.322993  2980 tracer.cc:136] Trace Op: fill_constant
I0518 15:32:40.322993  2980 layer.cc:357] Op(fill_constant): Inputs: ,   Outputs: Out{conv2d_1.b_0[LoDTensor<NOT_INITED>]}
I0518 15:32:40.322993  2980 prepared_operator.cc:104] expected_kernel_key:data_type[float]:data_layout[ANY_LAYOUT]:place[CUDAPlace(0)]:library_type[PLAIN]
I0518 15:32:40.322993  2980 layer.cc:386] Op(fill_constant): Inputs: ,   Outputs: Out{conv2d_1.b_0[LoDTensor<float, CUDAPlace(0), (16)>]}
I0518 15:32:40.323990  2980 tracer.cc:189] No Grad to track for Op: fill_constant
I0518 15:32:40.323990  2980 imperative.cc:543] Init VarBase
I0518 15:32:40.323990  2980 variable_wrapper.h:159] The tensor of variable linear_0.w_0 is not initialized
I0518 15:32:40.323990  2980 variable_wrapper.h:159] The tensor of variable linear_0.w_0 is not initialized
I0518 15:32:40.323990  2980 variable_wrapper.h:159] The tensor of variable linear_0.w_0 is not initialized
I0518 15:32:40.324987  2980 tracer.cc:136] Trace Op: uniform_random
I0518 15:32:40.327980  2980 layer.cc:357] Op(uniform_random): Inputs: ,   Outputs: Out{linear_0.w_0[LoDTensor<NOT_INITED>]}
I0518 15:32:40.327980  2980 prepared_operator.cc:104] expected_kernel_key:data_type[float]:data_layout[ANY_LAYOUT]:place[CUDAPlace(0)]:library_type[PLAIN]

还有些信息是训练开始后大量的输出，是一些关于变量操作的信息。

**Comment by shingstone-lu** [comment]
非常抱歉，源码搞错了。是下面这段：

import paddle
import paddle.vision.transforms as T
import paddle.nn.functional as F

paddle.set_device('gpu') # or 'cpu'
trainsform=T.Compose(
    [
        T.Normalize(
            mean=[127.5],
            std=[127.5],
            data_format='CHW'
        )
    ]
)
#加载数据集
train_dataset=paddle.vision.datasets.FashionMNIST(
    mode='train',transform=trainsform
)
test_dataset=paddle.vision.datasets.FashionMNIST(
    mode='test',transform=trainsform
)

#构建模型
class LeNet(paddle.nn.Layer):
    def __init__(self) -> None:
        super().__init__()

        self.conv1=paddle.nn.Conv2D(
            in_channels=1,out_channels=6,
            kernel_size=5,stride=1,padding=2
        )
        self.max_pool1=paddle.nn.MaxPool2D(
            kernel_size=2,stride=2
        )
        self.conv2=paddle.nn.Conv2D(
            in_channels=6,out_channels=16,
            kernel_size=5,stride=1
        )
        self.max_pool2=paddle.nn.MaxPool2D(
            kernel_size=2,stride=2
        )
        self.linear1=paddle.nn.Linear(
            in_features=16*5*5,out_features=120
        )
        self.linear2=paddle.nn.Linear(
            in_features=120,out_features=84
        )
        self.linear3=paddle.nn.Linear(
            in_features=84,out_features=10
        )
    
    def forward(self,inputs):
        y=self.conv1(inputs)
        y=F.relu(y)
        y=self.max_pool1(y)
        y=F.relu(y)
        y=self.conv2(y)
        y=self.max_pool2(y)
        y=paddle.flatten(y,start_axis=1,stop_axis=-1)
        y=self.linear1(y)
        y=F.relu(y)
        y=self.linear2(y)
        y=F.relu(y)
        y=self.linear3(y)

        return y
#模型配置
model=paddle.Model(LeNet())  #用Model封装模型
model.prepare(
    paddle.optimizer.Adam(
        learning_rate=0.001,
        parameters=model.parameters()
    ),
    paddle.nn.CrossEntropyLoss(),
    paddle.metric.Accuracy()
)
#模型训练
model.fit(
    train_dataset,
    epochs=50,
    batch_size=64,
    verbose=1
)
model.save('E:\\luqinghua\\LeNet\\model_train\\LeNet')  #保存模型 用于训练
model.save('E:\\luqinghua\\LeNet\\model_inference\\LeNet',training=False)  #保存模型 用于预测

**Comment by pangyoki** [comment]
你好，应该是默认开启了GLOG，可以使用 `export GLOG_v=0`来关掉GLOG。

**Comment by shingstone-lu** [comment]
我用VScode编辑的python文件，请问怎么关掉GLOG？

**Comment by shingstone-lu** [comment]
我在系统环境变量中找到了GLOG_v这个变量，值为10，我将其改为0然后重启系统之后没有上述大量中间信息的输出了！
非常感谢！
